pkg/scheduler/apis/config/register.go

```diff
@@ -45,6 +45,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&PodTopologySpreadArgs{},
 		&RequestedToCapacityRatioArgs{},
 		&ServiceAffinityArgs{},
+		&NodeResourcesLeastAllocatedArgs{},
```

Comment:
You have to do the same in `v1alpha2/register.go`

Please add tests in `pkg/scheduler/apis/config/scheme/scheme_test.go`

Reply:
done